pkgs/development/python-modules/polyline/default.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "polyline";
+  version = "1.4.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0x60lm3ccq9zrcqlzyk041xgr1mi0k9lzyyv3cbbdiq9kb88jzvw";
+  };
+
+  propagatedBuildInputs = [ six ];
+
+  # Tests have dependencies that are pinned to specific versions, which makes
+  # builds that run tests unreliable
+  doCheck = false;
+
+  meta = {
+    homepage = https://github.com/hicsail/polyline;
+    license = lib.licenses.mit;
+    description = "A Python implementation of Google's Encoded Polyline Algorithm Format.";
```

usually exclude articles
```suggestion
    description = "Python implementation of Google's Encoded Polyline Algorithm Format.";
```